mso/datasource_mso_schema_site_external_epg.go

```diff
@@ -76,7 +72,8 @@ func dataSourceMSOSchemaSiteExternalEpgRead(d *schema.ResourceData, m interface{
 		}
 		apiSiteId := models.StripQuotes(siteCont.S("siteId").String())
 
-		if apiSiteId == stateSiteId {
+		apiTemplate := models.StripQuotes(siteCont.S("templateName").String())
```

Comment:
Shall we use the !found flag above in the for loop as well? ( for i := 0; i < count && !found; i++)
Ref #120